src/schemas/fairdata-ida.ui.js

```diff
@@ -202,6 +202,18 @@ export default {
 		'title': "Creator of the dataset",
 		'description': "The principal researcher or researchers involved in producing the data.",
 	},
+	'/properties/creator/*/oneOf/0/properties/name': {
```

Comment:
why there is `0` and why these changes to schema?

Reply:
Person objects only appear under oneOf relation where index 0 is Person and index 1 is Organization. These changes added the description defined in the issue.

Reply:
ok